### PR TITLE
add citeas to downloaded lecture slides

### PR DESCRIPTION
**Pull request by RickardssonJ**

### CODE REVIEW

**Comment by tasola** [comment]
Building PR environment here: https://feature-ppt-add-citeas.d3188jlyh4w5pf.amplifyapp.com/ (not finished the build yet)

File: src/utils/createPptx/pptxConfiguration/slideElements.ts [hunk 1]
```diff
@@ -67,8 +67,8 @@ export const imageStyling: PptxGenJS.ImageProps = {
 
 export const citeAsStyling: PptxGenJS.TextPropsOptions = {
   x: '70%',
```
Comment (tasola) [inline src/utils/createPptx/pptxConfiguration/slideElements.ts:69]: The PR gets some build errors, so I can't try this out, but doesn't this look off? This places the citeAs to start at 70% of the slide's width, doesn't it?

File: src/utils/createPptx/pptxConfiguration/slideElements.ts [hunk 2]
```diff
@@ -67,8 +67,8 @@ export const imageStyling: PptxGenJS.ImageProps = {
...
-  y: '85%',
-  fontSize: 12,
+  y: '90%',
```
Comment (tasola) [inline src/utils/createPptx/pptxConfiguration/slideElements.ts:70]: Please express this with variables instead, probably break out mainContent's `h` to a variable MAIN_CONTENT_HEIGHT, and have this y-value be like Y_PADDING + MAIN_CONTENT_HEIGHT + CITE_AS_Y_MARGIN.

Also give this its own h; CITE_AS_HEIGHT and have it be remainingHeight(Y_PADDING * 2 + MAIN_CONTENT_HEIGHT + 2 * CITE_AS_Y_MARGIN)

(Take all values above with a grain of salt, it probably needs some fine-tuning)

**Review by tasola — CHANGES_REQUESTED** [review]

File: src/utils/createPptx/pptxConfiguration/slideElements.ts (hunk 1 above)
Comment (RickardssonJ) [reply src/utils/createPptx/pptxConfiguration/slideElements.ts:69]: Yes it puts it down in the right corner.
Should we keep it as a long string at the bottom instead?

File: src/utils/createPptx/pptxConfiguration/slideElements.ts (hunk 2 above)
Comment (RickardssonJ) [reply src/utils/createPptx/pptxConfiguration/slideElements.ts:70]: Done

**Pull request closed without merging**